jps-plugin/src/org/jetbrains/kotlin/compilerRunner/KotlinCompilerRunner.kt

```diff
@@ -117,9 +118,8 @@ public object KotlinCompilerRunner {
                 val stream = ByteArrayOutputStream()
                 val out = PrintStream(stream)
 
-// Uncomment after resolving problems with parallel compilation and tests
-//                if (System.getProperty(KOTLIN_COMPILER_ENVIRONMENT_KEEPALIVE_PROPERTY) == null)
-//                    System.setProperty(KOTLIN_COMPILER_ENVIRONMENT_KEEPALIVE_PROPERTY, "")
+                if (java.lang.Boolean.parseBoolean(System.getProperty(GlobalOptions.COMPILE_PARALLEL_OPTION, "false")))
```

Comment:
`String#toBoolean()` is just a Kotlin way of saying `Boolean.parseBoolean`, look at its implementation